backend/config/curation-api.yml

```diff
@@ -64,6 +64,22 @@ paths:
         - collection
       parameters:
         - $ref: "#/components/parameters/query_visibility"
+        - name: curator
```

Comment:
If `curator` is not specified, then the default is "self" ?

Editorial: Detailed information about the visibility parameter appears in the description for the endpoint. Should some of the curator examples move there? 

      description: >-
        * When the visibility parameter is unspecified or set to `PUBLIC`, a list of all public collections is returned.
        The Authorization header is not required. If a collection in the list has been deleted, then it is annotated
        with tombstone set to True.
        * When the visibility parameter is set to `PRIVATE`, a list of all private collections that the user is
        authorized to access is returned. The Authorization header is required. If a collection in the list is a private
        revision of a public collection, then it is annotated with revision_of.
        * Each collection in the list contains a subset of collection metadata as a preview.

Reply:
I think the default is `None`, and the backend defaults to returning all public Collections.

Which raises an interesting issue: it seems like the user would, if they supplied `curator_name="self"`, want to receive _all_ of their Collections, not just their public Collections.

Thoughts?

Reply:
> This API is targeted at _super curators_. Currently, they are authorized to see all collections, so a filter is proposed as a convenience.
> 
> The example below assume that a valid curator name is passed as the parameter.
> 
> Here's how I imagine the experience:
> 
> **User**
> 
> 1. GET /collections returns all the public collections.
> 2. GET /collections?curator_name="John Smith" returns all the public collections curated by John Smith. The use in this scenario is doubtful.
> 3. GET /collections?visibility="PRIVATE" returns UNAUTHORIZED.
> 4. GET /collections?visibility="PRIVATE"&curator_name="John Smith" returns UNAUTHORIZED.
> 
> **Curator**
> 
> 1. GET /collections returns all the public collections.
> 2. GET /collections?curator_name="John Smith" returns all the public collections curated by John Smith. Perhaps the curator is looking for best practices from another curator.
> 3. GET /collections?visibility="PRIVATE" returns all the curator's private collections.
> 4. GET /collections?visibility="PRIVATE"&curator_name="John Smith" returns UNAUTHORIZED if the curator is not "John Smith". Otherwise, it's equivalent to `3`. The use in this scenario is doubtful.
> 
> **Super Curator**
> 
> 1. GET /collections returns all the public collections.
> 2. GET /collections?curator_name="John Smith" returns all the public collections curated by John Smith.
> 3. GET /collections?visibility="PRIVATE" returns all private collections in the corpus.
> 4. GET /collections?visibility="PRIVATE"&curator_name="John Smith" returns all private collections in the corpus curated by John Smith.

I agree will all except Curator #4. While this does make logical sense to allow. It's tricky to do on the backend. For that reason, this scenario will return  UNAUTHORIZED. They should use `self` to get their own collections.